### PR TITLE
Switch to BYOC queues for internal builds

### PR DESCRIPTION
**Pull request by mmitche**

### CODE REVIEW

**Comment by mmitche** [comment]
Validated on an internal build

**Comment by MattGal** [comment]
Given what changed probably doesn't matter if you wait for CI...

File: azure-pipelines.yml [hunk 1]
```diff
@@ -27,7 +27,8 @@ jobs:
           name: NetCorePublic-Int-Pool
           queue: BuildPool.Windows.10.Amd64.VS2017.Open
         ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
-          name: dotnet-internal-temp
+          name: NetCoreInternal-Int-Pool
```
Comment (chcosta) [inline azure-pipelines.yml]: Random question.  What's the `-Int-` stand for?

Comment (mmitche) [reply azure-pipelines.yml]: it's effectively staging, since BYOC is still in preview.

**Pull request merged**